tests/UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/PriceChangeScenario.cs

```diff
@@ -14,7 +14,7 @@ public class PriceChangeScenarios
         private readonly string _projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory));
 
         //for local
-        // private readonly string _projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, "..\\..\\.."));
+        //private readonly string _projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, "..\\..\\.."));
```

Comment:
Remove commented code if not required.

Reply:
One path , is for pipeline and one is for local